src/Presentation/ListFacetHtmlBuilder.php

```diff
@@ -68,7 +71,7 @@ private function buildCheckboxesViewModel( FacetConfig $config, PropertyConstrai
 		foreach ( $this->getValuesAndCounts( $config ) as $i => $valueCount ) {
 			$checkboxes[] = [
 				'label' => $valueCount->value,
-				'count' => $valueCount->count, // FIXME: count is now showing in the UI for some reason
+				'count' => $valueCount->count,
 				'checked' => in_array( $valueCount->value, $selectedValues ), // TODO: test with multiple types of values
 				'value-id' => $valueCount->value,
 				// TODO: can't we escape this in the template?
```

Comment:
We can remove this TODO.
Mustache escaping is not sufficient for that since we need to escape whitespace as well.

Reply:
Is unrelated to this PR

The value is guaranteed to not contain spaces because we do not put anything in there that can have spaces. It's also guaranteed to "not need" escaping, but for security in layers, all computed values that are output are still escaped.

Reply:
So my current susicion is that we can remove the escaping here because mustache is already doing it. Needs verification first tho

Reply:
Yes Mustache already escapes HTML. If we don't have any whitespace in value then we can just drop Sanitizer and use Mustache directly.